docs/src/components/ReanimatedTestimonials/ReanimatedTestimonialList/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import ReanimatedTestimonialItem from '@site/src/components/ReanimatedTestimonials/ReanimatedTestimonialItem';
+
+const items = [
+  {
+    author: 'Marc Rousavy',
+    company: 'Margelo',
+    body: 'We’ve used Reanimated and Gesture Handler for a ton of apps already - it’s amazingly simple yet poweful! 😍',
+    link: 'https://twitter.com/mrousavy/status/1754909520571019756',
+    image: {
+      alt: 'marc rousavy',
+      src: 'img/testimonials/marc_rousavy.jpeg',
```

Comment:
```suggestion
      src: "https://pbs.twimg.com/profile_images/1713946397391667200/l-LXWNl2_400x400.jpg",
```

lets use remote images directly from Twitter